x/evm/client/cli/query.go

```diff
@@ -444,3 +444,38 @@ func GetCmdConfirmationHeight(queryRoute string) *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdGatewayTxState returns the query to get the minimum confirmation height for the given chain
+func GetCmdGatewayTxState(queryRoute string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gateway-tx-state [chain] [tx-id]",
```

Comment:
```suggestion
		Use:   "gateway-tx-state [chain] [txID]",
```
nitpick but I believe other commands use camel case for parameter names

Reply:
it's kinda all over the place but yeah :D